<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>GenerateOutliersDialog</class>
 <widget class="QDialog" name="GenerateOutliersDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>255</width>
    <height>273</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Generate Outliers Dialog</string>
  </property>
  <property name="windowIcon">
   <iconset resource="../res/Registar.qrc">
    <normaloff>:/images/bunny.png</normaloff>:/images/bunny.png</iconset>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QGridLayout" name="gridLayout">
     <item row="1" column="0">
      <layout class="QHBoxLayout" name="horizontalLayout_5">
       <item>
        <widget class="QLabel" name="label_2">
         <property name="text">
          <string>Ymin</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="yMinLineEdit">
         <property name="text">
          <string>-1.0</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item row="0" column="0">
      <layout class="QHBoxLayout" name="horizontalLayout_3">
       <item>
        <widget class="QLabel" name="label">
         <property name="text">
          <string>Xmin</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="xMinLineEdit">
         <property name="text">
          <string>-1.0</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item row="0" column="1">
      <layout class="QHBoxLayout" name="horizontalLayout_4">
       <item>
        <widget class="QLabel" name="label_7">
         <property name="text">
          <string>Xmax</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="xMaxLineEdit">
         <property name="text">
          <string>1.0</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item row="1" column="1">
      <layout class="QHBoxLayout" name="horizontalLayout_6">
       <item>
        <widget class="QLabel" name="label_4">
         <property name="text">
          <string>Ymax</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="yMaxLineEdit">
         <property name="text">
          <string>1.0</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item row="2" column="1">
      <layout class="QHBoxLayout" name="horizontalLayout_7">
       <item>
        <widget class="QLabel" name="label_6">
         <property name="text">
          <string>Zmax</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="zMaxLineEdit">
         <property name="text">
          <string>1.0</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item row="2" column="0">
      <layout class="QHBoxLayout" name="horizontalLayout_8">
       <item>
        <widget class="QLabel" name="label_3">
         <property name="text">
          <string>Zmin</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="zMinLineEdit">
         <property name="text">
          <string>-1.0</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_10">
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="boundingBoxButton">
       <property name="text">
        <string>Bounding Box</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QLabel" name="label_8">
       <property name="text">
        <string>num of points</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="numOfPointsLineEdit">
       <property name="text">
        <string>1</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_9">
     <item>
      <widget class="QLabel" name="label_5">
       <property name="text">
        <string>noise_std</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="noiseSTDLineEdit">
       <property name="text">
        <string>1.0</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QComboBox" name="methodComboBox">
     <item>
      <property name="text">
       <string>Random Distribution</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Random Along Normal</string>
      </property>
     </item>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <layout class="QGridLayout" name="gridLayout_2">
       <item row="0" column="0">
        <widget class="QPushButton" name="defaultButton">
         <property name="text">
          <string>Default</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QPushButton" name="helpButton">
         <property name="text">
          <string>Help</string>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QPushButton" name="closeButton">
         <property name="text">
          <string>Close</string>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QPushButton" name="applyButton">
         <property name="text">
          <string>Apply</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <widget class="QCheckBox" name="overwriteCheckBox">
         <property name="text">
          <string>overwrite</string>
         </property>
         <property name="checked">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="verticalSpacer">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
  <zorder>noiseSTDLineEdit</zorder>
  <zorder>layoutWidget</zorder>
  <zorder>methodComboBox</zorder>
  <zorder>label_8</zorder>
  <zorder>numOfPointsLineEdit</zorder>
  <zorder>boundingBoxButton</zorder>
  <zorder>horizontalSpacer</zorder>
 </widget>
 <resources>
  <include location="../res/Registar.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>closeButton</sender>
   <signal>clicked()</signal>
   <receiver>GenerateOutliersDialog</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>51</x>
     <y>184</y>
    </hint>
    <hint type="destinationlabel">
     <x>184</x>
     <y>190</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
